src/popup.ts

```diff
@@ -67,9 +67,13 @@ class MessageHandler {
 
 // To prevent clickjacking
 let allowPopup = window === window.top;
-window.addEventListener("message", async (e) => {
+window.addEventListener("message", async (e): Promise<void> => {
     if (e.source !== window.parent) return;
-    if (e.origin.endsWith('.youtube.com')) return allowPopup = true;
+    if (e.origin.endsWith('.youtube.com')) {
+        allowPopup = true;
+        return
+    }
+    else return;
```

Comment:
```suggestion
    if (e.origin.endsWith('.youtube.com')) {
        allowPopup = true;
    }
```

I think this would still be allowed